navigation/MainTabNavigator.js

```diff
@@ -141,16 +134,24 @@ export default createMaterialTopTabNavigator(
   },
   {
     tabBarPosition: 'bottom',
+    animationEnabled: false,
+    swipeEnabled: false,
+    lazy: true,
```

Comment:
lazy load views!

Reply:
Shouldn't this be `createMaterialBottomTabNavigator`? 
https://github.com/Phillip3/the-resonance-project/blob/487fd9462c191452690a590909bc8ae24efa33e1/navigation/MainTabNavigator.js#L127

I think we don't want to do lazy loading. We should load all of the tab's initial content before the app opens. Waiting on a loading indicator when you hit each tab is annoying.

Reply:
it seems to work fine right now, even with the hundreds of groups loading.

lazy loading is good because the user's device can focus on the the current tab and when they open the next tab, then we'll load whatever's needed

Reply:
i think we're refering to lazy-load for the visual after initial